### ISSUE TITLE
Move from 4 spaces to 2 spaces for indentation

### ISSUE THREAD

**Issue by smarella** [mesos/myriad#98]
A lot of open source java projects use 2 spaces for indentation.

**Comment by pdread100** [comment]
Why indent at all? Lets just use straight line code.

Keep it at 4 please.

**Comment by yufeldman** [comment]
I agree with Santosh. It is kind of a standard to keep 2 spaces for indentation - meaning your "tab" is 2 spaces
